Big_Integer/main.cpp

```diff
@@ -0,0 +1,476 @@
+#include<bits/stdc++.h>
+
+
+using namespace std;
+
+
+class Integer {
+    int sign;
+    string value;
+
+public:
+    Integer() {
+        //Default Empty Constructor::Dont use delegation...
+    }
+    //positive - negative value separator
+    Integer(string passed) {
```

Comment:
Codacy found an issue: [Class 'Integer' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6290669)